config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
   root 'users#index'
-
-  resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
-  end
+  get '/users', to: 'users#index', as: :users
+  get '/users:id', to: 'users#show', as: :user
+  get 'users:user_id/posts', to: 'posts#index', as: :user_posts
+  get 'users:user_id/posts:id', to: 'posts#show', as: :user_post
```

Comment:
[REQUIRED] To clarify the changes requested by the previous reviewer kindly ensure that your routes uses the appropriate syntax. Lines 5 and 6 should have slashes like below:
`get '/users/:id', to: 'users#show', as: :user
get '/users/:user_id/posts', to: 'posts#index', as: :user_posts
get '/users/:user_id/posts/:id', to: 'posts#show', as: :user_post`